deploy/k8s/chart/templates/_scc.tpl

```diff
@@ -0,0 +1,16 @@
+{{/*
```

Comment:
Right now the strategy is to:

* support all kinds of Kubernetes
* provide sane defaults
* allow overrides on different levels: global, module

I think the same can be applied here. So we could:

* use the full configuration instead of a boolean, the provided content only makes sense for OCP
* global -> module
* if this resolves to nothing:
  * if the instance is OCP: use the content that is now under "securityContextConstraintsEnabled"
  * if the instance is not OCP: use nothing

Additionally, the Helm charts tries not to use acronyms:

* `ocp` -> `openshift` (which we use in other locations too
* `scc` -> `securityContextConstraints` (which also is the name of the variable)